strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/RoutingRulesServiceImplTest.java

```diff
@@ -0,0 +1,166 @@
+        final Configuration configuration = configurationRepository.getConfiguration();
+
+        assertTrue(added);
+        configuration.getRoutingRules().getAccepted().get(GROUP_REPOSITORY).getRoutingRules().stream().filter(
+                routingRule -> routingRule.getPattern().equals(RULE_PATTERN))
+                     .forEach(routingRule -> assertTrue(routingRule.getRepositories().contains(REPOSITORY_ID_2)));
+    }
+
+    @Test
+    public void testRemoveAcceptedRepository()
+            throws Exception
+    {
+        routingRulesService.addAcceptedRuleSet(getRuleSet());
+
+        final boolean removed = routingRulesService.removeAcceptedRepository(GROUP_REPOSITORY, RULE_PATTERN,
+                                                                             REPOSITORY_ID);
+
+        final Configuration configuration = configurationRepository.getConfiguration();
+
+        configuration.getRoutingRules().getAccepted().get(GROUP_REPOSITORY).getRoutingRules().forEach(
+                routingRule -> {
+                    if (routingRule.getPattern().equals(RULE_PATTERN))
+                    {
+                        assertFalse(routingRule.getRepositories().contains(REPOSITORY_ID));
+                    }
+                }
+        );
+        assertTrue(removed);
+
+    }
+
+    @Test
+    public void testOverrideAcceptedRepositories()
+            throws Exception
+    {
+        routingRulesService.addAcceptedRuleSet(getRuleSet());
+
+        final RoutingRule rl = getRoutingRule();
+        final boolean overridden = routingRulesService.overrideAcceptedRepositories(GROUP_REPOSITORY, rl);
+        final Configuration configuration = configurationRepository.getConfiguration();
+        configuration.getRoutingRules().getAccepted().get(GROUP_REPOSITORY).getRoutingRules().forEach(
+                routingRule -> {
+                    if (routingRule.getPattern().equals(rl.getPattern()))
+                    {
+                        assertEquals(1, routingRule.getRepositories().size());
+                        assertEquals(rl.getRepositories(), routingRule.getRepositories());
+                    }
+                }
+        );
+
+        assertTrue(overridden);
+
+    }
+
+    private RoutingRule getRoutingRule()
+    {
+        RoutingRule routingRule = new RoutingRule();
+        routingRule.setPattern(RULE_PATTERN);
+        routingRule.setRepositories(new HashSet<>(Collections.singletonList(REPOSITORY_ID_2)));
+
+        return routingRule;
+    }
+
+    private RuleSet getRuleSet()
+    {
+        final RuleSet ruleSet = new RuleSet();
+        ruleSet.setGroupRepository(GROUP_REPOSITORY);
+        final RoutingRule routingRule = new RoutingRule();
+        routingRule.setPattern(RULE_PATTERN);
+        routingRule.setRepositories(new HashSet<>(Collections.singletonList(REPOSITORY_ID)));
+        ruleSet.setRoutingRules(Collections.singletonList(routingRule));
+        return ruleSet;
+    }
+}
```

Comment:
Please, add new lines at the end of files. Also, could you remove the empty lines at the start/end of methods? We usually leave an empty line before `return` statements.